notebooks/BikeShareExample.ipynb

```diff
@@ -468,8 +468,8 @@
     "                                                                              {\"FeaturizationMethodName\": \"filling\", \n",
```

Comment:
Do we need to keep Featurizations? If the user doesn't specify, it will be defaulted by the service.

Reply:
It is a required param for the call. Can't remove it.

Reply:
Sounds good. Will follow it up separately to see if it needs to be required?